release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -193,6 +193,24 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-8-bug-fixes"]
 == Bug fixes
 
+*Bare Metal Hardware Provisioning*
+
+* Previously, Ironic failed to download an image for installation because it uses HTTPS by default and Ironic does not have the correct certificate bundle available. This issue is fixed by setting the image download as insecure to request a transfer without the certificate. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1953795[*BZ#1953795*])
+
+* Previously, when using dual stack deployments, worker node hostnames sometimes did not match the name inspected before deployment. This caused nodes to need manual approval. This has been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1955114[*BZ#1955114*])
```

Comment:
s/dual stack deployments/dual-stack networking/

s/hostnames/host names/

"the name inspected before deployment" seems a little off.  Can you indicate what did the inspecting?..."...sometimes did not match the host name that <xyz> inspected/determined before installation."